webcompat/templates/home-page/browse-issues.html

```diff
@@ -2,16 +2,8 @@
   <div class="wc-UIContent">
     <h2 class="wc-Title wc-Title--l">Latest Untriaged Issues</h2>
     <div id="js-lastIssue">
-      <script type="text/template" id="needstriage-tmpl">
-        <% if (issues.length) { %>
-          <% _.each(issues, function(issue) { %>
-            {% include "web_modules/issue-list.html" %}
-          <% }); %>
-        <% } else { %>
-        <p>Nothing here currently, cool!</p>
-      <% } %>
-      </script>
-      </div>
+      {% include "web_modules/issue-list.jst" %}
```

Comment:
Note: we don't actually need to include this template here in the HTML, since it effectively does nothing, but I thought we should consider keeping it so it's less confusing to know what templates are used where. Or, we could change this to an HTML comment which means less work for the jinja parser.